Configure

```diff
@@ -2673,6 +2673,7 @@ make)
 	gmake)
 	echo "I can't find make or gmake, and my life depends on it." >&4
 	echo "Go find a public domain implementation or fix your PATH setting!" >&4
```

Comment:
Not sure if it'd be good to replace "public domain" with "open source" as strictly speaking there might be no PD implementation of such tools at all.